pkg/odo/cli/pipelines/utility/utility.go

```diff
@@ -35,3 +38,47 @@ func MaybeCompletePrefix(s string) string {
 	}
 	return s
 }
+
+// Client represents a client for K8s
+type Client struct {
+	KubeClient kubernetes.Interface
+}
+
+// NewClient returns a new K8s client
+func NewClient(client kubernetes.Interface) *Client {
+	return &Client{KubeClient: client}
+}
+
+// CheckIfSealedSecretsExists checks if sealed secrets is installed
+func (c *Client) CheckIfSealedSecretsExists(ns, name string) error {
```

Comment:
making this a `types.NamespacedName` makes it less likely to make mistakes.

Reply:
Updated to use `types.NamespacedName`